### PR TITLE
Fix for docmentation bug on RenderObject.parentData

### PR DESCRIPTION
**Pull request by slightfoot**
I don't think a description is required.. such a minor change.  
Changed "sector layout" for "sliver layout".

### CODE REVIEW

File: packages/flutter/lib/src/rendering/object.dart [hunk 1]
```diff
@@ -1051,7 +1051,7 @@ abstract class RenderObject extends AbstractNode with DiagnosticableTreeMixin im
   ///    calling [setupParentData] on the future parent node.
   ///  * The conventions for using the parent data depend on the layout protocol
   ///    used between the parent and child. For example, in box layout, the
-  ///    parent data is completely opaque but in sector layout the child is
+  ///    parent data is completely opaque but in sliver layout the child is
```
Comment (Hixie) [inline packages/flutter/lib/src/rendering/object.dart]: Actually sector layout was intentional. It refers to:
https://github.com/flutter/flutter/blob/master/examples/layers/rendering/src/sector_layout.dart

Comment (Hixie) [reply packages/flutter/lib/src/rendering/object.dart]: (In sliver layout, the child is not permitted to read fields either.)

**Comment by Hixie** [comment]
Maybe we can turn this into a link to the relevant GitHub file?

File: packages/flutter/lib/src/rendering/object.dart (hunk 1 above)
Comment (slightfoot) [reply packages/flutter/lib/src/rendering/object.dart]: I'll close this PR then? seems' pretty pointless now ;)

**Pull request closed without merging**